subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -190,6 +192,7 @@ static int download_client_callback(const struct download_client_evt *event)
 			}
 
 			send_progress((offset * 100) / file_size);
+			downloading = false;
```

Comment:
This should be removed as it is not working, if the Download progress is enabled

Reply:
I tought i removed it ... must have pushed the wrong

Reply:
Now it's good